### PR TITLE
Added multi-currency support for magento 2.0

### PR DESCRIPTION
**Pull request by mayankamencherla**
Magento 2.x will now support multiple currencies.

### CODE REVIEW

File: Controller/Payment/Order.php [hunk 1]
```diff
@@ -64,11 +64,14 @@ public function execute()
             if(null !== $order && !empty($order->id))
```
Comment (73sl4) [inline Controller/Payment/Order.php]: space after if

File: Controller/Payment/Order.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ public function __construct(
 
     public function execute()
     {
-        $amount = (int) (round($this->getQuote()->getGrandTotal(), 2)*100);
+        $amount = (int) (round($this->getQuote()->getBaseGrandTotal(), 2)*100);
```
Comment (73sl4) [inline Controller/Payment/Order.php]: space after and before `*`

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js [hunk 3]
```diff
@@ -92,6 +92,10 @@ define(
                     email: billing_address.email
                 };
 
+                if (quote.totals()['quote_currency_code'] !== 'INR'){
```
Comment (73sl4) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: remove this block.

Comment (73sl4) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: Change L86 to this.amount = quote.totals()['base_grand_total']*100;

File: Controller/Payment/Order.php (hunk 2 above)
Comment (mayankamencherla) [inline Controller/Payment/Order.php]: changed.

File: Controller/Payment/Order.php (hunk 1 above)
Comment (mayankamencherla) [inline Controller/Payment/Order.php]: changed.

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js (hunk 3 above)
Comment (mayankamencherla) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: Done.

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js [hunk 4]
```diff
@@ -171,7 +171,14 @@ define(
                         contact: this.user.contact,
                         email: this.user.email
                     }
-                });
+                };
+
+                if (data.quote_currency !== 'INR'){
```
Comment (73sl4) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: opening bracket in next line.

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js [hunk 5]
```diff
@@ -107,7 +107,7 @@ define(
                         self.handleError(result);
                     }
                 );
-
+
```
Comment (73sl4) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: remove trailing whitespaces.

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js (hunk 4 above)
Comment (mayankamencherla) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: done

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js (hunk 5 above)
Comment (mayankamencherla) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: done

**Pull request merged**